packages/components/src/toggle-group-control/toggle-group-control-option-base/styles.ts

```diff
@@ -64,6 +64,30 @@ export const buttonView = ( {
 		border: 0;
 	}
 
+	&[disabled],
+	&[aria-disabled='true'] {
+		cursor: default;
+		${ ButtonContentView } {
+			opacity: 0.4;
+		}
+
+		/**
+		 * Show an additional focus ring for disabled radio items, since the
+		 * indicator won't follow keyboard focus. This is not an issue for non-radio
+		 * items, since the backdrop is already decoupled from keyboard focus.
+		 */
+		&[role='radio']:focus-visible::after {
```

Comment:
The focus ring is applied to the `after` pseudo-element so that we can control its size via the `inset` prop, so that we can leave a little gap between the focus ring and the backdrop indicator.